samples/openapi3/client/petstore/python/tests/test_configuration.py

```diff
@@ -76,5 +76,14 @@ def test_get_host_from_settings(self):
         self.assertEqual("http://petstore.swagger.io:8080/v2", self.config.get_host_from_settings(0, {'port': '8080'}))
         self.assertEqual("http://dev-petstore.swagger.io:8080/v2", self.config.get_host_from_settings(0, {'server': 'dev-petstore', 'port': '8080'}))
 
+    def testConfigurationDebug(self):
+        for debug, expected in [(True, True), (False, False), (None, False)]:
+            with self.subTest(debug=debug, expected=expected):
+                c = petstore_api.Configuration(debug=debug)
+                self.assertEqual(expected, c.debug)
+        with self.subTest(expected=False):
+            c = petstore_api.Configuration()
+            self.assertEqual(expected, c.debug)
```

Comment:
I think this works a bit by chance, since `expected` in the assert is the one taken from the for-loop line 80.

Can you make the test a bit more explicit?

```suggestion

        expected = False
        with self.subTest(debug="unset", expected=expected):
            c = petstore_api.Configuration()
            self.assertEqual(expected, c.debug)
```

Reply:
d'oh, you're right!
rewrote them and added explicit message